app/cni/pkg/cni/main.go

```diff
@@ -85,13 +91,43 @@ func parseConfig(stdin []byte) (*PluginConf, error) {
 	return &conf, nil
 }
 
+func hijackMainProcessStderr() *os.File {
+	file, err := getCniProcessStderr()
+	if err != nil {
+		log.Error(err, "could not hijack main process file - continue logging to "+defaultLogLocation)
+		return nil
+	}
+	log.V(0).Info("successfully hijacked stderr of cni process - logs will be available in 'kubectl logs'")
+	os.Stderr = file
+	log = core.NewLoggerTo(os.Stderr, 2).WithName(defaultLogName)
+	return file
+}
+
+func getCniProcessStderr() (*os.File, error) {
+	pids, err := pidOf("/install-cni")
+	if err != nil {
+		return nil, err
+	}
+	if len(pids) != 1 {
+		return nil, errors.New("more than one process '/install-cni' running on a node, this should not happen")
+	}
+
+	file, err := os.OpenFile(path.Join("/", "proc", strconv.Itoa(pids[0]), "fd", "2"), os.O_WRONLY, 0)
+	return file, err
```

Comment:
maybe
```suggestion
	return os.OpenFile(path.Join("/proc", strconv.Itoa(pids[0]), "/fd/2"), os.O_WRONLY, 0)
```
?
(it will also be consistent with https://github.com/kumahq/kuma/pull/4846/files#diff-048e7ce6fe8f663c75e13096784c7bbfaf5990aef555ee44c10f7ee25da014c9R58 in regards of the path)

Reply:
that's the release branch, so let's move the discussion about it in different PR